apps/guide/src/content/03-creating-your-bot/01-configuration-files.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Configuration files
+category: Creating your bot
+---
+
+# Configuration files
+
+Once you [add your bot to a server](/preparations/adding-your-bot-to-servers.md), the next step is to start coding and get it online! Let's start by creating a config file to prepare the necesssary values your client will need.
+
+As explained in the ["What is a token, anyway?"](/preparations/setting-up-a-bot-application.md#what-is-a-token-anyway) section, your token is essentially your bot's password, and you should protect it as best as possible. This can be done through a _`config.json`_ file or by using environment variables.
```

Comment:
```suggestion
As explained in the ["What is a token, anyway?"](../installations-and-preparations/setting-up-a-bot-application.md#what-is-a-token-anyway) section, your token is essentially your bot's password, and you should protect it as best as possible. This can be done through a _`config.json`_ file or by using environment variables.
```